Builds/DockerPlay/README.md

```diff
@@ -0,0 +1,157 @@
+# DockerPlay : what is this ?
+
+As a lucky Debian Jessie users some of us are missing an impotant stack to play with rippled from the sources : gcc5. Instead of hacking our working station (and potentially break it) we can setup a rippled development environment thanks docker. 
+
+Basic idea: being able to edit rippled source files with our favorite tools and then build, test and run rippled inside docker containers. 
+
+# What do you need to play ?
```

Comment:
Prefer "What do you need to use this container?" instead of "What do you need to play ?" Similarly, below, in the "How to play ?" prefer "How to use" instead.